en/docs/apis/create-and-publish-your-first-api.md

```diff
@@ -52,34 +48,34 @@ Follow this procedure to test the API revision that you deployed:
 
     ![API GET resource](../assets/img/apis/api-resource.png){.cInlineImage-full}
     
-4. Click **Try it out** and then click **Execute**. You can see the COVID-19 totals for all US states as the response from the API.
+4. Click **Try it out** and then click **Execute**. You can see the COVID-19 data for all the states of the USA as the response from the API.
 
-    ![Response received from the service execution](../assets/img/apis/response.png){.cInlineImage-full}
+    ![Response received from the service execution](../assets/img/apis/api-response.png){.cInlineImage-full}
 
 Now you are ready to publish the API to the API Developer Portal so that external consumers can consume the API.
 
 ## Step 4: Publish the API
-Follow this procedure to publish the API and view the published API on the API Developer Portal:
+Follow this procedure to publish and view the API to the API Developer Portal so that external consumers can consume the API:
 
 1. Click **Publish** in the left pane.
 2. On the **Lifecycle Management** page, click **Publish**.
 3. Click **Go to Developer Portal**. This signs you in to the API Developer Portal.
 
-    ![Go to API Developer Portal](../assets/img/apis/go-to-devportal.png){.cInlineImage-full}
+    ![Go to API Developer Portal](../assets/img/apis/go-to-developer-portal.png){.cInlineImage-full}
      
     You can see the published API in the API Developer Portal.
     
-    ![List of published APIs in the API Developer Portal](../assets/img/apis/devportal.png){.cInlineImage-full}
+    ![List of published APIs in the API Developer Portal](../assets/img/apis/developer-portal.png){.cInlineImage-full}
 
 ## Step 5: Invoke the API 
 Follow this procedure to invoke the API from the API Developer Portal:
 
 1. Click **Credentials** in the left pane.
-2. On the **Credentials** page, click **Generate Credentials**. This creates credential for an internal application and subscribes this API to that internal application.
+2. Click **Generate Credentials**. This creates credential for an internal application and subscribes this API to that internal application.
```

Comment:
"creates credential" should change to "create credentials".

Reply:
```suggestion
2. Click **Generate Credentials**. This creates credentials for an internal application and subscribes this API to that internal application.
```